iceoryx2-bb/lock-free/tests/spmc_unrestricted_atomic_tests.rs

```diff
@@ -108,3 +108,19 @@ fn spmc_unrestricted_atomic_load_store_works_concurrently() {
         });
     });
 }
+
+#[test]
+fn spmc_unrestricted_atomic_get_ptr_write_and_update_works() {
```

Comment:
Can you add another concurrent tests. Two threads, one thread reads the value in an infinite busy loop while the other one is using this API to write an incrementing large `u128` number in there.

When some memory order is wrong miri (whenever we activate her) should be able to detect it.